app/Package.php

```diff
@@ -119,14 +119,26 @@ public function toSearchableArray()
         // Temporarily truncate to prevent algolia from throwing a size exceeded exception
```

Comment:
Does this same limitation exist in Typesense? Even if it doesn't, it might be a worthwhile practice anyway, but then update the comment to remove "algolia"

Reply:
This is a great catch. Turns out Typesense doesn't have this limitation so we're removing it.